holidays/countries/united_states.py

```diff
@@ -234,6 +262,15 @@ def _add_subdiv_al_holidays(self):
         if self._year >= 1890:
             self._add_holiday_1st_mon_of_jun("Jefferson Davis Birthday")
 
+        # Columbus Day / American Indian Heritage Day / Fraternal Day
+        if self._year >= 1971:
+            name = (
+                "Columbus Day / American Indian Heritage Day / Fraternal Day"
+                if self._year >= 2000
+                else "Columbus Day / Fraternal Day"
+            )
+            self._add_holiday_2nd_mon_of_oct(name)
```

Comment:
```suggestion
            self._add_holiday_2nd_mon_of_oct(
                "Columbus Day / American Indian Heritage Day / Fraternal Day"
                if self._year >= 2000
                else "Columbus Day / Fraternal Day"
            )
```

well, here and after where name is going to be used only once